src/create.rs

```diff
@@ -41,16 +58,20 @@ impl Create {
             bail!("{} already exists", self.container_id)
         }
 
+        // change directory to the bundle directory, and load configuration,
+        // copy that to the container's directory
         unistd::chdir(&self.bundle)?;
 
         let spec = oci_spec::Spec::load("config.json")?;
         fs::copy("config.json", container_dir.join("config.json"))?;
         log::debug!("spec: {:?}", spec);
 
+        // convert path to absolute path
```

Comment:
It might be better to indicate the reason here.
The reason for converting to an absolute path is that a path relative to where the youki command is executed will be difficult to manipulate using that relative path when chdir is used.